src/SprykerEco/Zed/AmazonQuicksight/Communication/Controller/UserController.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AmazonQuicksight\Communication\Controller;
+
+use Spryker\Zed\Kernel\Communication\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Csrf\CsrfToken;
+
+/**
+ * @method \SprykerEco\Zed\AmazonQuicksight\Persistence\AmazonQuicksightRepositoryInterface getRepository()
+ * @method \SprykerEco\Zed\AmazonQuicksight\Business\AmazonQuicksightFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\AmazonQuicksight\Communication\AmazonQuicksightCommunicationFactory getFactory()
+ */
+class UserController extends AbstractController
+{
+    /**
+     * @uses \Spryker\Zed\AnalyticsGui\Communication\Controller\AnalyticsController::indexAction()
+     *
+     * @var string
+     */
+    protected const URL_ANALYTICS = '/analytics-gui/analytics';
```

Comment:
Can we make not hardcoded redirect but redirect to the previous page? e.g redirect back should redirect to the page where button was located?

Reply:
Fixed for UserController, for AnalyticsController this is not possible since there is a confirmation page but we need to redirect to the main one